src/pages/imageSetup/ImageSetup.tsx

```
@@ -1,4 +1,4 @@
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
```

ESLint

[error] Replace `'react'` with `"react"`

(prettier/prettier)